src/components/transport_manager/src/websocket_server/websocket_session.cc

```diff
@@ -108,6 +112,8 @@ TransportAdapter::Error WebSocketSession<ExecutorType>::WriteDown(
     return TransportAdapter::FAIL;
   }
 
+  on_data_sent_(message);
```

Comment:
@mvorobio 
Need we use the DataSendFailed in the negative case, if we use the DataSendDone ?

Reply:
@SNiukalov  please check https://github.com/LuxoftSDL/sdl_core/pull/18/commits/7bc1f2ce4c04d80778a833d7022995d806ce5788